### ISSUE TITLE
Pointer speed setting does nothing

### ISSUE THREAD

**Issue by dusssk12** [linearmouse/linearmouse#806]
### OS

macOS 15.0 (Build 24A335)

### LinearMouse

v0.10.0

### Describe the bug

When I disable mouse acceleration with the button, the pointer is too fast even with speed set to 0 and moving the slider does nothing. When I leave acceleration on and turn it off by setting it to 0 with the slider, then the pointer is way too slow and again, changing the speed slider doesn't do anything. This happened when I updated to MacOS 15 so it might have to do something with that because it wasn't happening before.

### To reproduce

1. Try disabling mouse acceleration with either the slider or the on/off button on OS15 2. try changing pointer speed and see if it does anything

**Comment by kurtistowery** [comment]
I have the same issue

**Comment by snoolord** [comment]
my trackpad speed is really slow after macos sequoia update. i think it must require a fix for macos 15.

**Comment by Slackow** [comment]
> my trackpad speed is really slow after macos sequoia update. i think it must require a fix for macos 15.

same issue

**Comment by ulgens** [comment]
+1 for the speed issue. Please let us know if there is any manual fix.

**Comment by fmflurry** [comment]
Pointer speed has no effect on any of my mouse anymore too.

**Comment by DamCongDuy** [comment]
My cursor speed is even slower with LinearMouse enabled.
macOS Sequoia 15.0.1 (24A348)
v0.10.0

**Comment by jwang-paradise** [comment]
It's not as bad as "doing nothing", but is indeed much much slower than on macOS 14.
Hope developer can look into this. It's making my life very painful right now lol.

**Comment by lujjjh** [comment]
Thanks for reporting this! I’m not seeing this issue on macOS 15, so it’d be great if you could share your configuration file, any relevant logs, and the model of the mouse you're using. That info will help me dig deeper into the issue and find a fix.

**Comment by DamCongDuy** [comment]
Send you my configuration and logs files. Hope the problem will be fixed soon :D

[linearmouse.json](https://github.com/user-attachments/files/17350628/linearmouse.json)
[LinearMouse.log](https://github.com/user-attachments/files/17350629/LinearMouse.log)

**Comment by jwang-paradise** [comment]
It seems that **some values in the config are clipped** within a certain range (e.g. 20 for "acceleration", 1 for "speed").

For example, when I set `"acceleration" : 21.11110` (_larger than 20_) in the config, the program clipped it to 20 (see log below).

config (part):
```
{
  "pointer" : {
    "acceleration" : 21.1111,
    "speed" : 0.119,
    "disableAcceleration" : false
  }
}
```

log shows:
```
2024-10-13 04:25:48 +0000	info	EventTransformerManager	Initialize EventTransformer with scheme: 
{"scrolling":{"modifiers":{"command":{"scale":2,"type":"changeSpeed"}},"speed":{"vertical":0},"acceleration":{"vertical":1},"distance":"auto"},"pointer":{"acceleration":20,"speed":0.119,"disableAcceleration":false},"buttons":{"universalBackForward":false},"if":{"app":"com.lujjjh.LinearMouse","display":"Built-in Retina Display","device":{"productName":"Apple Internal Keyboard \/ Trackpad","category":"trackpad","vendorID":"0x5ac","productID":"0x342"},"groupApp":"com.lujjjh.LinearMouse"}} (device=Optional(Apple Internal Keyboard / Trackpad (VID=0x05AC, PID=0x0342)), pid=Optional(94796), screen=Optional("Built-in Retina Display"))
```
Note the pointer object: `"pointer":{"acceleration":20,"speed":0.119,"disableAcceleration":false}`
The "speed" is correct as specified in the config, but "acceleration" is _only 20_.

---
I'm not sure if the range is required by macOS tho. Hope this info is of some help!

**Comment by cuboooooo** [comment]
+1 to this bug! MacOS 15.0.1, Linear Mouse only decelerates the pointer when the settings tab is active, and the pointer speed has little effect. Reinstalling and restarting did not fix. Seems to be a Sequoia issue.

**Comment by andrewbraxton** [comment]
Pointer options work fine for me with my Orochi V2 mouse, but none of the options do anything for the internal trackpad or external Magic Trackpad on Sequoia 15.0.1. Let me know if there is any diagnostic information you need.

**Comment by lujjjh** [comment]
There is a known issue where the pointer speed setting doesn’t work on built-in trackpads starting from M2 models and also doesn’t function on the Magic Trackpad, starting with macOS Sequoia. This issue also affects tools like CursorSense.

It appears challenging to resolve, as macOS has removed higher-level support, requiring us to look for more low-level solutions.

I’d like to continue collecting feedback beyond the situations mentioned above. If you’re unable to adjust pointer speed for your mouse in macOS Sequoia, please attach your config and logs.

**Comment by kurtistowery** [comment]
I have already reverted back to macOS sonoma but adjusting my mouse speed using LinearMouse on Sequoia 
didn't work. I'm going to attach my log and config file anyway case it would be of help. 

[linearmouse.json](https://github.com/user-attachments/files/17560840/linearmouse.json)
[LinearMouse.log](https://github.com/user-attachments/files/17560846/LinearMouse.log)

**Comment by YoShLKSL** [comment]
I am encountering the same problem after upgrading to "Sequoia." I've temporarily switched to CursorSense. I'm hopeful that a solution will be found shortly. I have attached the necessary files. I am utilizing the built-in trackpad.

[linearmouse.json](https://github.com/user-attachments/files/17683449/linearmouse.json) 

[LinearMouse.log](https://github.com/user-attachments/files/17683446/LinearMouse.log)

**Comment by fogx** [comment]
same issue, but my config for scrolling and mouse are both completely reset after the update, so nothing to share really

**Comment by dkmar** [comment]
Built-in trackpad and the ceiling for pointer speed seems really low. I would love to make it faster than the max so I suspect it isn't working. Seems pretty indistinguishable from what's made available in System Settings.

[linearmouse.json](https://github.com/user-attachments/files/17709413/linearmouse.json)
[LinearMouse.log](https://github.com/user-attachments/files/17709415/LinearMouse.log)

**Comment by dkmar** [comment]
> I've temporarily switched to CursorSense. 

@YoShLKSL Does CursorSense work on sequoia for trackpad? Their [FAQ](https://plentycom.jp/en/cursorsense/support.html) seems to imply it doesn't work.

**Comment by lamnguyenx** [comment]
I second this. And btw the default value seems so weird:
<img width="962" alt="Screenshot 2024-11-21 at 11 24 46" src="https://github.com/user-attachments/assets/69a13af7-7692-4c8a-b114-c4c07f41d928">

**Comment by ulgens** [comment]
The issue is still valid.

**Comment by cantonalex** [comment]
:( My index finger is sore

**Comment by ulgens** [comment]
The issue is not stale.

**Comment by XDGFX** [comment]
Yep same issue here. If I check the "Disable pointer acceleration" and use the new Tracking speed slider that seems to work.

**Comment by ulgens** [comment]
>  If I check the "Disable pointer acceleration" and use the new Tracking speed slider that seems to work.

Tested with 0.10.1-beta.2, didn't create any difference in tracker speed 🙁

**Comment by ulgens** [comment]
Not stale.

**Comment by ulgens** [comment]
This issue still exist. Can we somehow exempt this from stale bot?

**Comment by patrickrushton** [comment]
Same problem here. Also, with LinearMouse running, the tracking speed slider in macOS Tahoe Settings has no effect. So I am able to get a good balance of pointer speed and acceleration.

**Comment by fuweichin** [comment]
I know that v0.10.4 is trying to "Fix tracking speed not working for Trackpad and Magic Mouse".

Since the "Tracking speed" option doesn't work for Magic Mouse when "Disable pointer acceleration" is checked, it's recommended to hide that option for Magic Magic before the issue gets fixed.

Why I need to change tracking speed of magic mouse
+ Magic Mouse 2 operates at a native, fixed DPI of 1300\~1600, while I have accustomed to pointer sensitivity of 1000\~1200 dpi for image editing.
+ The pointer sensitivity gap between Magic Mouse and Trackpad distracts me.

On Windows, I can disable pointer acceleration for mouse by unchecking "Enhance pointer precision", I can disable pointer acceleration for both mouse and window precision touchpad by using [The MarkC Windows 10 + 8.1 + 8 + 7 Mouse Acceleration Fix](http://donewmouseaccel.blogspot.com/2010/03/markc-windows-7-mouse-acceleration-fix.html). in the mean while I can still change pointer move speed.


macOS 15.7 arm64
